tests/e2e/index.js

```diff
@@ -0,0 +1,19 @@
+const p = require('phin');
+const Evervault = require('../../lib');
+
+const evervault = new Evervault(process.env.EV_API_KEY, {
+  debugRequests: true,
+  enableOutboundRelay: true,
+});
+
+evervault.encrypt('I should be decrypted').then((result) => {
+  const res = p({
+    url: process.env.TARGET_URL,
+    method: 'POST',
+    data: {
+      decryptedField: result,
+    },
+    parse: 'json',
+  });
+  console.log(res.body);
+});
```

Comment:
was this meant to be included? We should be making an assertion on the res body at least

Reply:
Was going to include but removed because I cant assert the value of `TARGET_URL` if I don't set it